Assets/Scripts/Framework/Gameplay/Interactions/Interactable.cs

```diff
@@ -53,6 +54,9 @@ public abstract class Interactable : NetworkBehaviour
         [Server]
         public bool InteractAsServer(Interactor interactor) => Interact(interactor);
 
+        [Server]
+        public Collider[] GetCollidingColliders() => Physics.OverlapBox(BoxCollider.center + transform.position, BoxCollider.size);
```

Comment:
```suggestion
        public Collider[] GetCollidingColliders() 
                => Physics.OverlapBox(BoxCollider.center + transform.position, BoxCollider.size);
```
Line exceeds 120 chars